kart/json_diff_writers.py

```diff
@@ -226,9 +231,37 @@ def write_header(self):
                 "outputFormat": "JSONL+hexwkb",
             }
         )
+        if self._diff_estimate_accuracy is not None:
+            t = threading.Thread(
+                target=self._calculate_and_feature_count_estimate,
+                # don't block process exit.
+                daemon=True,
```

Comment:
Not sure this is safe; e.g. it might cause the annotations database to get corrupted or locked. second opinion @olsen232 ? Using a non-daemon thread is safer but we'll need to build some kind of alerting to kill the thread on process exit :/

Reply:
Removed this in favour of a `threading.Event` and the thread terminating itself.